### ISSUE TITLE
Resolving videos for some playlists does not work

### ISSUE THREAD

**Issue by markushl** [Tyrrrz/YoutubeExplode#518]
For some playlists (example: https://www.youtube.com/watch?v=VI4ssGtfdxw&list=RDVI4ssGtfdxw) the retrieval of contained videos doesn't work.

In YoutubeExplode.ReverseEngineering.Responses.PlaylistResponse.GetAsync() at \YoutubeExplode\ReverseEngineering\Responses\PlaylistResponse.cs(209) with scrapePage = true the URL https://www.youtube.com/playlist?list=RDEMPeuXuiddECzwmszjRj1EtQ&hl=en&persist_hl=1 will be downloaded, but the response (checked in browser) returns the error "This playlist type is unviewable."

How can this be fixed?

Edit: Other example is https://www.youtube.com/watch?v=7QWHKFcjP_0&list=RDCMUCFaGFYoav4S3eGk1Z55Kmww

**Comment by Tyrrrz** [comment]
I had no idea such unviewable playlists were a thing.

**Comment by System0x20** [comment]
I also have a similar problem but with YouTube Music.
Well I found out that YouTube Music songs are kinda normal YouTube videos/audios
For example for
https://music.youtube.com/watch?v=OsfAnsMY21M
Just change music to www like this
https://www.youtube.com/watch?v=OsfAnsMY21M (Dua Lipa - Levitating btw)
I tried to `get_video_info` it. But `status` entry from `playabilityStatus` of `player_response` is `UNPLAYABLE`
Thus it doesnt even give me streams to play with their signature and dash nonsense.
